pkg/decoder/decoder.go

```diff
@@ -7,12 +7,14 @@ package decoder
 
 import (
 	"bytes"
+	"regexp"
+	"time"
 
 	"github.com/DataDog/datadog-log-agent/pkg/config"
 	"github.com/DataDog/datadog-log-agent/pkg/message"
 )
 
-// Payload represents a list of bytes and an optional reference to its origin
+// Payload represents a list of bytes coming from a file and an optional reference to its origin
```

Comment:
this is not true, payload can come from docker or from the network.

However, could we add a comment to offset?
```
  offset int64 // when set, the exact position in the file where the data comes from
```